firebase_insert_block.py

```diff
@@ -7,7 +7,7 @@
 @discoverable
 class FirebaseInsert(FirebaseBase):
 
-    version = VersionProperty("1.0.0")
+    version = VersionProperty("2.0.0")
```

Comment:
This is a recent development as we're just now releasing blocks to the api, but it is no longer required to manually bump the version properties in the block files, spec files, or release files. Rather once this PR is merged, you can do a jenkins command to automatically bump the versions for these blocks (you'll see jenkins actually has a commit on here doing just that, releasing V1.0.0.). So any versions that are bumped here should be left as the previous value.

Reply:
I'll show you the jenkins process or I can just bump it after this is merged since I'm doing a bunch of jenkins stuff today anyways.

Reply:
That would be great. Should I revert the version back to 1.0.0 for these blocks in the spec and release json files as well?

Reply:
Yes every version in this repo should be `1.0.0` at this point